src/core/core/rpc_engine.cpp

```diff
@@ -154,29 +154,16 @@ namespace dsn {
             // server address side effect
             auto req = call->get_request();
             auto sp = task_spec::get(req->local_rpc_code);
-            if (reply->from_address != req->to_address)
+            // TODO(qinzuoyan): because current rpc forward is fake (resend from client), we cannot
```

Comment:
(1) reply->from_address is slways equal to req->to_address, because  in rpc_engine::call_ip() there is a line "request-to_address = addr".
(2) so I add TODO here, in case that when other cases added, we should refactor code structure in a systematic way.